tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {Component, Input, ViewChild} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {
+  ColumnHeaders,
+  SelectedStepRunData,
+} from '../../metrics/views/card_renderer/scalar_card_types';
```

Comment:
One small thing that keeps bothering me is, importing `scalar_card_types` in a widget, both here and data_table_component. This kind of makes the widget not so widget. Have you considering putting these types in 'data_table_types' and import in scalar_card_component? 

This does not need to be included in this PR.